watchtower/src/main.rs

```diff
@@ -158,9 +159,13 @@ fn get_config() -> Config {
 }
 
 fn process_confirmed_block(notifier: &Notifier, slot: Slot, confirmed_block: ConfirmedBlock) {
-    let break_program_id = "BrEAK7zGZ6dM71zUDACDqJnekihmwF15noTddWTsknjC"
-        .parse::<Pubkey>()
-        .unwrap();
+    let break_program_ids: HashSet<Pubkey> = [
+        "BrEAK7zGZ6dM71zUDACDqJnekihmwF15noTddWTsknjC",
+        "brkLHMdveYjzgaK3a4FbofAGhj3QJRAUyNxLJqKsKXs",
```

Comment:
This is the account pubkey, not the program ID I believe, so may not match on `program_pubkey` but I'm not 100%

Reply:
Yeah, `brk..` is a system account used to create other accounts. To filter those transactions, we need to check it against all of the message account keys